CMakeLists.txt

```diff
@@ -85,9 +85,10 @@ endif()
 #
 find_package(PkgConfig)
 pkg_check_modules(GST
+	REQUIRED
```

Comment:
Actually gstreamer-1.0 is not required package - it is optional. If gstreamer and related packages are not found then QGC should build without video streaming support (and it is how it builds using qmake, see .pro/.pri files). If you want to get it merged you need to keep it working properly - make gstreamer optional and compile with video disabled if related packages were not found.